odoo/tools/config.py

```diff
@@ -117,6 +117,9 @@ def __init__(self, fname=None):
         group.add_option("--addons-path", dest="addons_path",
                          help="specify additional addons paths (separated by commas).",
                          action="callback", callback=self._check_addons_path, nargs=1, type="string")
+        group.add_option("-M", "--migrations-path", dest="migrations_path",
```

Comment:
I would not support any short option, because of the way optparse allows partial parameter names - it increases the chance of conflicts.